openslides_backend/action/actions/motion/create_base.py

```diff
@@ -0,0 +1,106 @@
+import time
+from typing import Any, Dict
+
+from ....models.models import Motion
+from ....shared.exceptions import ActionException
+from ....shared.patterns import Collection, FullQualifiedId
+from ...mixins.create_action_with_dependencies import CreateActionWithDependencies
+from ..agenda_item.agenda_creation import CreateActionWithAgendaItemMixin
+from ..agenda_item.create import AgendaItemCreate
+from ..list_of_speakers.create import ListOfSpeakersCreate
+from ..list_of_speakers.list_of_speakers_creation import (
+    CreateActionWithListOfSpeakersMixin,
+)
+from ..motion_submitter.create import MotionSubmitterCreateAction
+from .sequential_numbers_mixin import SequentialNumbersMixin
+from .set_number_mixin import SetNumberMixin
+
+
+class MotionCreateBase(
+    CreateActionWithDependencies,
+    CreateActionWithAgendaItemMixin,
+    SequentialNumbersMixin,
+    SetNumberMixin,
+    CreateActionWithListOfSpeakersMixin,
+):
+
+    model = Motion()
+    dependencies = [AgendaItemCreate, ListOfSpeakersCreate]
+
+    def set_state_from_workflow(
+        self, instance: Dict[str, Any], meeting: Dict[str, Any]
+    ) -> None:
+        workflow_id = instance.pop("workflow_id", None)
+        if workflow_id is None:
+            if instance.get("lead_motion_id"):
+                workflow_id = meeting.get("motions_default_amendment_workflow_id")
+            elif instance.get("statute_paragraph_id"):
+                workflow_id = meeting.get(
+                    "motions_default_statute_amendment_workflow_id"
+                )
+            else:
+                workflow_id = meeting.get("motions_default_workflow_id")
+        if workflow_id:
+            workflow = self.datastore.get(
+                FullQualifiedId(Collection("motion_workflow"), workflow_id),
+                ["first_state_id"],
+            )
+            instance["state_id"] = workflow.get("first_state_id")
+        else:
+            raise ActionException(
+                "No matching default workflow defined on this meeting"
+            )
+
+    def check_for_origin_id(self, instance: Dict[str, Any]) -> None:
```

Comment:
Move this directly to `create_forwarded.py` since it is only needed there and a special thing for forwarded motions. the regular `motion.create` should not know about it.

Reply:
Moved `check_for_origin_id()` to motion.create_forwarded.